File: Assignment_15/assignment5.c
/////////////////////////////////////////////////////////
//
//  File name :     assignment5.c
//  Description :   Accept number of rows and columns from user and display below pattern.
//  Author :        Aryan S. Pardeshi.
//  Date :          04/06/2025
//
/////////////////////////////////////////////////////////

// Input  :  iRow = 4   iCol = 4
// Output :  1  2  3  4  
//           1  *  *  4
//           1  *  *  4  
//           1  2  3  4  

#include<stdio.h>

#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i = 0;
    int j = 0;

    if (iRow < 0)
    {
        iRow = -iRow;
    }
    if (iCol < 0)
    {
        iCol = -iCol;
    }

    for (i = 1; i <= iRow; i++)
    {
        for (j = 1; j <= iCol; j++)
        {
            if ((i == 1) || (i == iRow) || (j == 1) || (j == iCol))
            {
                printf("%d\t",j);
            }
            else
            {
                printf("*\t");
            }
        }
        printf("\n");
    }
}

int main() 
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter the number of Rows and Columns: ");
    scanf("%d %d", &iValue1, &iValue2);

    Pattern(iValue1, iValue2);

    return 0;
}

File: Assignment_44/assignment5.java
/////////////////////////////////////////////////////////////////////////////////////////////
//
//   File name   :  assignment5.java
//   Description :  Write a program to accept the matrix and swap the contents of consecutive rows.
//   Autor name  :  Aryan Shailendrasingh Pardeshi
//   Date        :  02/08/2025
//
////////////////////////////////////////////////////////////////////////////////////////////

/*
  Input :

  3 2 5 9
  4 3 2 2
  8 4 1 5
  3 9 7 5

  Output : 

  4 3 2 2
  3 2 5 9
  3 9 7 5
  8 4 1 9
*/

import java.util.*;

class Matrix
{
    private int Arr[][];

    public Matrix(int A, int B)
    {
        Arr = new int[A][B];
    }

    public void Accept()
    {
        System.out.println("Please Enter the elements of matrix");

        Scanner sobj = new Scanner(System.in);
        int i = 0, j = 0;

        for(i = 0; i < Arr.length; i++)
        {
            for(j = 0; j < Arr[i].length; j++)
            {
                Arr[i][j] = sobj.nextInt();
            }
        }
    }

    public void Display()
    {
        System.out.println("Elements of the matrix are : ");

        int i = 0, j = 0;

        for(i = 0; i < Arr.length; i++)
        {
            for(j = 0; j < Arr[i].length; j++)
            {
                System.out.print(Arr[i][j]+"\t");
            }
            System.out.println();
        }
    }
    
    public void SwapRows()
    {

    }
}

class assignment5
{
    public static void main(String A[])
    {
        int iRet = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number of rows : ");
        int iRow = sobj.nextInt();

        System.out.println("Enter the number of coloums : ");
        int iCol = sobj.nextInt();

        Matrix mobj = new Matrix(iRow, iCol);

        mobj.Accept();

        mobj.Display();

        mobj.SwapRows();

    }
}